builder/builder.go

```diff
@@ -2260,7 +2260,7 @@ New upstream: %d (format: %s)
 	b.UpstreamVerUint32 = nextUpstream
 	b.UpstreamVer = nextUpstreamStr
 
-	if _, err := b.CheckBumpNeeded(); err != nil {
+	if _, err := b.CheckBumpNeeded(false); err != nil {
```

Comment:
Erm.. Perhaps I'm misunderstanding, but if you call this with `false` here, the user will not be warned that their new state is bad and they need to fix it. Is this a deliberate choice, pushing the warning to the next time they try to do a build?